tiled/src/test/kotlin/ktx/tiled/MapLayerTest.kt

```diff
@@ -44,4 +45,34 @@ class MapLayerTest {
   fun `should not retrieve non-existing property from MapLayer`() {
     mapLayer.property<String>("non-existing")
   }
+
+  @Test
+  fun `should return true when MapLayers is empty`() {
+    val actual = MapLayers()
+
+    assertTrue(actual.isEmpty())
+  }
+
+  @Test
+  fun `should return false when MapLayers is not empty`() {
+    val actual = MapLayers()
+    actual.add(MapLayer())
+
+    assertFalse(actual.isEmpty())
+  }
```

Comment:
This might be stretching it a bit and testing the internal implementation, but if there's a possibility of removing a layer, I'd also add a test where you add and remove the layer to see if the count is correctly updated and the extension methods work properly.

I feel like some additional tests that are borderline for LibGDX itself are a good thing, considering they don't have much of a test suite as far as I know.

Reply:
I added that test case. I don't know if people ever add/remove layers manually. The way I use it is that the entire map from Tiled gets loaded with the TmxMapLoader of LibGDX. That loader prepares a TiledMap and all its content and I never manipulated the map data during runtime.

However, it is possible and maybe people out there do it, so it makes sense to have a test for that.

Reply:
Thanks.